frontend/src/components/common/form/PlusButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
I suspect we'll want to reorganize where we keep buttons.  Maybe they belong in a `/components/common/button` folder

Reply:
If this component is going in the top level components/common/form area it should try and be generic. This component is quite specific to the add agency page, so in its current form it should be located under something like /features/adminAgencies/components.

If you want to leave this here, strip out all the feature specific text/logic and make them settable via params (in this case just the tooltip)

Reply:
Sounds good!